doc/repository-spec-v2.md

```diff
@@ -22,22 +40,24 @@ repository.
 
 **Optional headers** used for popularity metrics.
 
-* `X-Pub-OS: windows | macos | linux | ...`
```

Comment:
Please avoid unrelated changes in the PR. Do them separately. It makes it hard to review!

Reply:
sorry for this; it's just vscode that changed the * into - for markdown. I can revert if you want.

Reply:
I suggest we avoid canonicalization, especially when it depends on the vscode plugin in use :D

Reply:
Reverted commit